src/bun.js/api/server.zig

```diff
@@ -3240,7 +3240,7 @@ fn NewRequestContext(comptime ssl_enabled: bool, comptime debug_mode: bool, comp
                                 var prev = body.value.Locked.readable;
                                 body.value.Locked.readable = .{};
                                 readable.value.ensureStillAlive();
-                                prev.deinit();
+                                defer prev.deinit();
```

Comment:
In the future some kind of lint could help us with this. It's easy to miss a `defer`.

Reply:
@cirospaciari are you sure this fixes anything? readable.value.ensureStillAlive() should make sure its visible to GC


> In the future some kind of lint could help us with this. It's easy to miss a `defer`.

pretty hard to tell when it's necessary

Reply:
hm we did do this for fetch as well